pkg/api/vlabs/validate.go

```diff
@@ -16,13 +16,26 @@ import (
 var (
 	validate        *validator.Validate
 	keyvaultIDRegex *regexp.Regexp
+	// Any version has to be mirrored in https://acs-mirror.azureedge.net/github-coreos/etcd-v[Version]-linux-amd64.tar.gz
+	etcdValidVersions = [...]string{"2.5.2", "3.2.0"}
 )
 
 func init() {
 	validate = validator.New()
 	keyvaultIDRegex = regexp.MustCompile(`^/subscriptions/\S+/resourceGroups/\S+/providers/Microsoft.KeyVault/vaults/[^/\s]+$`)
 }
 
+func isValidEtcdVersion(etcdVersion string) error {
+	validVersions := ""
```

Comment:
"" is a valid version, because it will be set to default when defaults are set.  Ensure "" is allowed.